tutorial/docker_config/slurm/Dockerfile

```diff
@@ -10,8 +10,11 @@ RUN curl -o miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-L
 ENV PATH /opt/anaconda/bin:$PATH
 RUN conda install --yes -c conda-forge python=3.8 dask distributed flake8 pytest pytest-asyncio
 RUN conda install --yes -c conda-forge openmpi mpi4py
-RUN git clone https://github.com/E-CAM/jobqueue_features.git
-
+RUN git clone https://github.com/E-CAM/jobqueue_features.git --branch temp_branch 
```

Comment:
Ignore the rest of this PR, it is just commits from other PRs for testing. This line is the important one, we should be able to pass in an argument so that we can check out a particular branch but it will take a bit of work to figure it out. I don't plan on doing it right now, but if someone wants to they can leverage this workflow

Reply:
The other commits are for #78 and #77 which I wanted to test. They work!